data_warehouse/services/aggregate_data_service.py

```diff
@@ -380,3 +387,42 @@ def table_training_feedback(cls, context):
             )
         file_path = TableExportService.export_training_feedback(data)
         return file_path, '培训反馈表.xls'
+
+    @classmethod
+    def table_training_records(cls, context):
+        '''个人培训记录导出'''
+        request = context.get('request')
+        event_name = context.get('event_name', None)
```

Comment:
No need to use `context.get`, `context['xxx']` is enough, since your serializer has already provided the default values for all fields.